### PR TITLE
Unit description rework

### PR DESCRIPTION
**Pull request by overdramaticpanromantic**
- Reworks all T1, T2, T3, and mothership descriptions
- I tried to keep all details via use of, well, details.
- Also slightly changes core unit descriptions.

### CODE REVIEW

File: content/units/t3/springald.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "type": "tank",
   "name": "[orange]Springald",
-  "description": "A large tank equipped with a rocket launching cannon and small laser arrays. Excels at combat versus buildings but is at a slight disatvantage versus units.",
+  "description": "Equipped with a rocket launching cannon and small laser arrays. Excels at siege against buikldings, but is weaker against units.",
```
Comment (Catana791) [inline content/units/t3/springald.json]: Typo

File: content/units/t3/plant.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "type": "payload",
   "name": "[green]Sapling",
-  "description": "A large support drone with decent health and very high healing capabilities.",
+  "description": "Capable of large-scale mining, builing, and repairing. The penultimate support drone.",
```
Comment (Catana791) [inline content/units/t3/plant.json]: Typo

File: content/units/mothership/test.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "[purple]Lycosid",
   "type": "legs",
-  "description": "A slow but very strong spider-like unit equipped with highly deadly weapons!",
+  "description": "Equipped with elemental cannons. Constructs and deploys toructs and reducts.",
```
Comment (Catana791) [inline content/units/mothership/test.json]: Capitalization of unit names

File: content/units/mothership/sps.json [hunk 4]
```diff
@@ -1,7 +1,8 @@
 {
   "type": "flying",
   "name": "[orange]Orbital Platform Strahl",
-  "description": "The orbital platform Strahl is one of the most secretive units every made. The unit is known for its destructive potential and deadly range yet it never once was mentioned in any record. While the technologies that the Strahl has seemingly pioneered are being implemented by military factions across the cluster, few will ever know that the unit truly existed. Fewer still will have ever encountered it and, the few that did will have not survived. May be flying but is so heavy that it can't fly over water without getting destroyed.",
+  "description": "Boasts several highly concentrated laser beams, alongside point defense turrets. Rapidly builds and releases Alopex.",
```
Comment (Catana791) [inline content/units/mothership/sps.json]: Needs a small change

File: content/units/t3/plant.json (hunk 2 above)
Comment (overdramaticpanromantic) [reply content/units/t3/plant.json]: GHAHHHHHHAHDKSADJSADS

File: content/units/mothership/sps.json [hunk 5]
```diff
@@ -1,7 +1,8 @@
...
+  "details": "One of the most secretive units ever built. Few have witnessed its power, albeit its technology being used commonly in modern-day military efforts across the cluster. Beware!
```
Comment (Catana791) [inline content/units/mothership/sps.json]: Forgot ",

File: content/units/mothership/hive-attack.json [hunk 6]
```diff
@@ -1,7 +1,8 @@
 {
   "type": "flying",
   "name": "[purple]HIVE Heavy Assault Unit (HHAU)",
-  "description": "Spawned by the HIVE mothership and equipped with special sensors that allow it to always keep the perfect distance to the enemy, this drone attacks your enemies in the most optimal ways",
+  "description": "Equipped a high-velocity missile launcher.",
+  "details": "Has many sensors that operate off enemy positions, allowing it to stay an optimal distance away at all times. Highly reliable assault unit.
```
Comment (Catana791) [inline content/units/mothership/hive-attack.json:5]: Forgot ",

File: content/units/mothership/corax.json [hunk 7]
```diff
@@ -1,7 +1,7 @@
 {
   "type": "legs",
   "name": "[green]Corax",
-  "description": "A heavy unit with an appearance similar to that of Corvus. Shoots a similar laser to Corvus but also some light artillery.",
+  "description": "Charges up a gigawatt laser that obliterates enemy structures and units alike. Additionally equipped with smaller, artillery lasers.",
```
Comment (Catana791) [inline content/units/mothership/corax.json:4]: We don't need a comma here.

**Review by Catana791 — APPROVED** [review]

**Pull request merged**